README.md

```diff
@@ -36,6 +36,19 @@ Once you have them you could try building them:
 cargo run --package fontc -- --source ../google_fonts_sources/sources/ofl/notosanskayahli/sources/NotoSansKayahLi.designspace
 ```
 
+## Contributing
+
+We have included a few git hooks that you may choose to use to ensure that
+patches will pass CI; these are in `resources/githooks`.
+
+If you would like to have these run automatically when you commit or push
+changes, you can set this as your git hooksPath:
+
+```sh
+git config core.hooksPath "resources/githooks"
+```
+
```

Comment:
Optional: Suggest adding something akin to "or just run resources/githooks/pre-push" manually when you are ready to push. Handy for people like me who want to choose when it runs.

Reply:
added!